lesson_10/test_homework_10.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from unittest.mock import patch, MagicMock
+import logging
+from homework_10 import log_event
+
+
+class TestLogEvent(unittest.TestCase):
```

Comment:
It is better further to use pytest framework for development your test cases. It is more flexible and more popular among Test Automation community.

Reply:
Thanks for the advice